### PR TITLE
New Ubuntu Server appliance

### PR DESCRIPTION
**Pull request by msxy22**
Before submitting a pull request, please check the following.

---
When updating an **existing** appliance:
- [ ] The new version is on top.
- [ ] The filenames in the "images" section are unique, to avoid appliances / version overwriting each other.
- [ ] If you forked the repo, running check.py doesn't drop any errors for the updated file.
---
When creating a **new** appliance:
- It's tested locally, i.e.
  - [ ] You dragged an instance into a project on your box, got it installed (if necessary), and did some basic network checks (ping, UI reachable, etc.).
  - [ ] GNS3 VM can run it without any tweaks.
  - [ ] The device is in the right category: router, switch, guest (hosts), firewall
  - [ ] You filled in as much info as possible (checks the schemas and other appliance files for some guidance).
- [ ] When adding a container: it builds on Docker Hub and can be pulled.
- [ ] The filenames in the "images" section are unique (to avoid appliances and/or versions overwriting each other).
- [ ] If you forked the repo, running check.py doesn't drop any errors for the new file.
- [ ] *Optional: a symbol has been created for the new appliance.*

### CODE REVIEW

**Comment by grossmj** [comment]
Looks good however my only concern is the name of the appliance because the vdmk file is not distributed by an official vendor (Canonical). Is the Ubuntu installation customized? Also, the MD5 for this file is not `ca131c8da907e6e60f4be92c3aecac29` but `435f15a54f7f673e302ad26f05226e0e`

File: ubuntu-server-18.04.3 (64bit) LTS.gns3a [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "vendor_name": "Canonical Inc.",
+    "vendor_url": "https://www.ubuntu.com",
+    "documentation_url": "https://help.ubuntu.com",
+    "product_name": "Ubuntu",
+    "product_url": "https://ubuntu.com/server",
+    "registry_version": 3,
+    "status": "stable",
+    "maintainer": "GNS3 Team",
+    "maintainer_email": "developers@gns3.net",
+    "usage": "Username: gns3\nPassword: gns3 | MD5: ca131c8da907e6e60f4be92c3aecac29",
```
Comment (msxy22) [inline ubuntu-server-18.04.3 (64bit) LTS.gns3a]: "usage": "Username: gns3\nPassword: gns3 | MD5: 435f15a54f7f673e302ad26f05226e0e",

**Comment by msxy22** [comment]
Any news ?

**Comment by grossmj** [comment]
Sorry it took so long. Thanks for your contribution! :)

**Comment by msxy22** [comment]
> Sorry it took so long. Thanks for your contribution! :)

- Well you should take a holiday from time to time and for the contribution well I have been following GNS3 I think since 2008-2009 😄.
- Just a question when will it be available in the marketplace ?

**Comment by grossmj** [comment]
I do take holidays from time to time ;)

I posted the appliance on the marketplace:
https://gns3.com/marketplace/appliance/ubuntu-server

**Pull request merged**